psst-gui/src/ui/playlist.rs

```diff
@@ -245,5 +362,46 @@ fn playlist_menu(playlist: &Playlist) -> Menu<AppState> {
         .command(cmd::COPY.with(playlist.url())),
     );
 
+    if library.contains_playlist(playlist) {
+        if library.is_created_by_user(playlist) {
+            let unfollow_msg = UnfollowPlaylist {
+                link: playlist.link(),
+                created_by_user: true,
+            };
```

Comment:
This can be moved up with `created_by_user` being set to `library.is_created_by_user(playlist)`